CHANGELOG.md

```diff
@@ -3,6 +3,74 @@
 ## EdgeX Foundry Services
 [Github repository](https://github.com/edgexfoundry/edgex-go)
 
+## [v2.1.0] Jakarta - 2021-11-17 (Compatible with the 2.0.0 release)
```

Comment:
I used `Not Compatible with 1.x releases` in other changes logs, but now thinking it should be `(Only compatible with the 2.x releases)